app/src/androidTest/java/com/github/polybooks/database/SaleDatabaseTest.kt

```diff
@@ -245,4 +246,36 @@ class SaleDatabaseTest {
         future = db.querySales().getN(1, 0)
         assertTrue(future.get().size <= 1)
     }
+
+    @Test
+    fun addDelete(){
+        val saleTest = Sale("test-tqwjdhsfalkfdhjasdhlfkahdfjklhdjhfl.adfjasdhflka-adjklshfjklasdhfjklhasd",
+            301943, 666f,
+            BookCondition.WORN,
+            Timestamp(com.github.polybooks.database.format.parse("2016-05-05")!!),
+            SaleState.RETRACTED )
+        db.deleteSale(saleTest)
+        BaristaSleepInteractions.sleep(2000, TimeUnit.MILLISECONDS)
+        assertEquals(0,db.querySales().searchByTitle(saleTest.title).getCount().get())
+        db.addSale(saleTest)
+        BaristaSleepInteractions.sleep(2000, TimeUnit.MILLISECONDS)
+        assertEquals( listOf(saleTest), db.querySales().searchByTitle(saleTest.title).getAll().get())
+        db.deleteSale(saleTest)
+        BaristaSleepInteractions.sleep(2000, TimeUnit.MILLISECONDS)
+        assertEquals(0,db.querySales().searchByTitle(saleTest.title).getCount().get())
+
+
+    }
+    
+    @Test
+    fun Delete(){
```

Comment:
This doesn't test anything which isn't tested by fun addDelete()? Why does it exist?

Reply:
To manually add and delete values from firebase